Tools/tracedump.cc

```diff
@@ -216,17 +216,20 @@ inline void print_register(const debug_trace_entry &e)
 	}
 }
 
+#ifdef SIGINFO
```

Comment:
This doesn't need to be conditional on SIGINFO, it will just be dead code.  Mark the variable and function `static` and they'll be optimised away if not used.